packages/cli/lib/lib/webpack/webpack-base-config.js

```diff
@@ -182,10 +183,14 @@ module.exports = function createBaseConfig(env) {
 					loader: require.resolve('babel-loader'),
 					options: Object.assign(
 						{ babelrc: false },
-						createBabelConfig(env, { browsers }),
+						createBabelConfig(env, {
+							browsers: info.compiler.name === 'InjectManifest'
```

Comment:
what exactly does this do?

Reply:
It's supposed to keep modern syntax in both generated `sw.js` outputs, since service worker is only supported by fairly modern browsers with generator/async support anyway.

Reply:
ah, right

Reply:
maybe it would make sense to add a comment for future reference